Evgenii/style.css

```diff
@@ -0,0 +1,501 @@
+    justify-content: center;
+    align-items: center;
+}
+.title{
+    font-weight: 700;
+    font-size: 36px;
+    color: white;
+    text-transform: uppercase;
+    margin-bottom: 27px;
+    white-space: nowrap;
+} 
+.main{
+    width: 665px;
+    height: 245px;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+
+}
+.main_text{
+    width: 410px;
+    height: 50px;
+    display: flex;
+    justify-content: space-between;
+    text-align: center                       ;
+    font-size: 18px;
+    color: #fff;
+    font-weight: 400;
+}
+.btn_buttons{
+    white-space: nowrap;
+}
+.reg{
+    margin-right: 15px;
+    padding: 12px 26px 13px;
+    border-radius: 3px;
+    background: transparent;
+    border: 1px solid;
+    font-size: 20px;
+    color: #fff;
+    font-weight: 600;
+    background: #00be2b;
+    border: #00be2b;
+    text-align: center;
+}
+.entrance{
+    background-color: rgba(0,0,0,.3);
+    border-color: #00be2b;
+    padding: 12px 26px 13px;
+    border-radius: 3px;
+    background: transparent;
+    border: 1px solid;
+    font-size: 20px;
+    color: #fff;
+    font-weight: 600;
+    text-align: center;
+}
+.entrance:hover{
+    background: #00be2b;
+    transition: all .3s;
+}
+.reg:hover{
+    background: #30cd73;
+    transition: all .3s;
+}
+.cur{
+    width: 25px;
+    height: 52px;
+    position: absolute;
+    top: 60%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+    background-repeat: no-repeat;
+    cursor: pointer;
+}
+.grid_conteiner{
+    width: 100%;
+    display: grid;
+    grid-template-columns: 1fr 360px 360px 360px 1fr;
+    grid-template-rows: 200px 200px 150px 100px;
+    grid-column-gap: 85px;
+}
+.grid_conteiner > *
+{
+    padding: 0px 0px 30px;
+    display: grid;
+    align-items: center;
+    justify-items: center;
+}
+.services__title{
+    text-transform: uppercase;
+    padding: 85px 0px 30px;
+    text-align: center;
+    font-size: 42px;
+    color: #2acd73;
+    grid-column: 2/5;
+    grid-row: 1/2;
+}
+hr{
+    width: 65px;
+    background: #30cd73;
+    height: 3px;
+    margin-top: 26px;
+}
+.services__item-title{
+    color: #000;
+    font-size: 22px;
+    font-weight: 500;
+    text-align: center;
+}
+.services__item-text{
+    font-weight: 400;
+    font-size: 15px;
+    color: #a7afb8;
+    text-align: center;
+    padding-bottom: 50px;
+}
+.services__item-one{
+    grid-column: 2/3;
+    grid-row: 2/3;
+    text-align: center;
+}
+.services__item-two{
+    grid-column: 3/4;
+    grid-row: 2/3;
+    text-align: center;
+
+}
+.services__item-three{
+    grid-column: 4/5;
+    grid-row: 2/3;
+    text-align: center;
+}
+.services__item-four{
+    grid-column: 2/3;
+    grid-row: 3/4;
+    text-align: center;
+}
+.services__item-five{
+    grid-column: 3/4;
+    grid-row: 3/4;
+    text-align: center;
+}
+.services__item-six{
+    grid-column: 4/5;
+    grid-row: 3/4;
+    text-align: center;
```

Comment:
text-align inherited property, specify once to its parent.
https://developer.mozilla.org/ru/docs/Web/CSS/text-align